cmd/src/campaigns_new.go

```diff
@@ -102,23 +102,28 @@ func getGitConfig(attribute string) (string, error) {
 	return strings.TrimSpace(string(out)), nil
 }
 
-const campaignSpecTmpl = `name: hello-world-campaign
-description: This campaign adds Hello World to READMEs
+const campaignSpecTmpl = `name: NAME-OF-YOUR-CAMPAIGN
+description: DESCRIPTION-OF-YOUR-CAMPAIGN
 
-# Find all repositories that contain a README.md file.
+
+# "on" specifies on which repositories to execute the "steps"
 on:
+  # Example: find all repositories that contain a README.md file.
   - repositoriesMatchingQuery: file:README.md
 
-# In each repository, run this command. Each repository's resulting diff is captured.
+# steps are run in each repository. Each repository's resulting diff is captured.
```

Comment:
This feels inconsistent with the other comments in the template. I'm not going to hold up today's release for this, but I'll open a PR to play around with this wording some more.